versioned_docs/version-2.6/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/migrate-rancher-to-new-cluster.md

```diff
@@ -52,7 +52,7 @@ Install the [`rancher-backup chart`](https://github.com/rancher/backup-restore-o
 
      The above assumes an environment with outbound connectivity to Docker Hub.
 
-     For an **air-gapped environment**, use the Helm value below to pull the `backup-restore-operator` image from your private registry when installing the rancher-backup Helm chart.
+     For an **air-gapped environment**, set a `$REGISTRY` variable to point to the path to your registry. Then, use the following Helm value to pull the `backup-restore-operator` image from your private registry when you install the rancher-backup Helm chart.
 
      ```bash
      --set image.repository $REGISTRY/rancher/backup-restore-operator
```

Comment:
```suggestion
     For an **air-gapped environment**, use the following `image.repository` Helm value to pull the `backup-restore-operator` image from your private registry when you install the rancher-backup Helm chart. Replace `<registry>` with  your registry URL.

     ```bash
     --set image.repository <registry>/rancher/backup-restore-operator
```

I'm thinking `$REGISTRY` in the original example command was intended to be a placeholder and not an actual variable? There's only a single occurrence on this page so using a variable adds extra steps to do the same thing. If we want to keep it a variable though, then the example should be like what line 43 did and have

```
REGISTRY=<registry>
```

Reply:
Probably just simpler to use `<registry>`